src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static final int MIN_AGE = 18;
+
+    @BeforeAll
+    public static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_NullLogin_ThrowsException() {
```

Comment:
sometimes you have 'ThrowsException', sometimes you have 'NotOk'. Be consistent in naming.
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/b53387ea-57a9-4f91-a724-13f19289204a)